packages/create-carbon/template/tsconfig.json.hbs

```diff
@@ -0,0 +1,32 @@
+---
+path: tsconfig.json
+---
+
+{
+    "include": ["src"{{#if (eq runtime "nextjs")}}, "next-env.d.ts", ".next/types/**/*.ts"{{/if}}],
+    "exclude": ["node_modules"{{#if (eq runtime "nextjs")}}, ".next"{{else}}, "dist"{{/if}}],
+    "compilerOptions": {
+		"strict": true,
+		"skipLibCheck": true,
+        {{#if (eq runtime "nextjs")}}
+		"lib": ["dom", "dom.iterable", "esnext"],
+        {{else}}
+        "lib": ["es2022"],
+        {{/if}}
+        "baseUrl": ".",
+        "outDir": "dist",
+        {{#if (eq runtime "nextjs")}}
+        "noEmit": true,
+		"module": "esnext",
+		"moduleResolution": "bundler",
+		"allowJs": true,
+		"resolveJsonModule": true,
+		"esModuleInterop": true,
+		"isolatedModules": true,
+        "plugins": [{ "name": "next" }],
+        "jsx": "preserve",
+		"incremental": true,
+        {{/if}}
+        "paths": { "~/*": ["src/*"] }
+    }
```

Comment:
There's some funky indentation going on in this file

Reply:
Can you clarify? As far as I can tell the intention is fine in nextjs and other runtimes generated files

Reply:
It's a mixture of both spaces and tabs, instead of one or the other (tabs >>>>)